ziggurat-core-geoip/Cargo.toml

```diff
@@ -11,9 +11,9 @@ description = "GeoIP localization package for Ziggurat"
 [dependencies]
 async-trait = "0.1.63"
 geoutils = "0.5.1"
-ip2location = "0.3.5"
+ip2location = "0.4.2"
```

Comment:
A general file comment:

Let's not specify the patch version of the SemVer, no need for that.
And it will make maintenance easier
```suggestion
ip2location = "0.4"
```